api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/NewCardService.java

```diff
@@ -4,22 +4,28 @@
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.factory.CardRequestFactory;
+import uk.gov.dhsc.htbhf.claimant.model.card.CardRequest;
 import uk.gov.dhsc.htbhf.claimant.repository.ClaimRepository;
 
+import java.util.UUID;
+import javax.persistence.EntityNotFoundException;
+
 @Service
 @AllArgsConstructor
 @Slf4j
 public class NewCardService {
 
+    private CardClient cardClient;
+    private CardRequestFactory cardRequestFactory;
     private ClaimRepository claimRepository;
 
-    /**
-     * Print out first 10 new claims (testing purposes only).
-     */
     @Transactional
-    public void createNewCards() {
-        claimRepository.getNewClaims().limit(10).forEach(
-                claim -> log.info("Creating card for new claim {}", claim.getId())
-        );
+    public void createNewCards(UUID claimId) {
+        Claim claim = claimRepository.findById(claimId)
+                .orElseThrow(() -> new EntityNotFoundException("Unable to find claim with id " + claimId));
+        CardRequest cardRequest = cardRequestFactory.createCardRequest(claim);
+        cardClient.createNewCardRequest(cardRequest);
```

Comment:
We should add event logging here (or maybe in the method calling this), but I'm happy to leave that and do it as part of HTBHF-941

Reply:
Added TODO